osu.Game/Rulesets/Difficulty/DifficultyCalculator.cs

```diff
@@ -37,5 +37,20 @@ protected virtual void PreprocessHitObjects()
         }
 
         public abstract double Calculate(Dictionary<string, double> categoryDifficulty = null);
+
+        protected double StrainCalculator(List<double> highestStrains, double decayWeight)
```

Comment:
`CalculateStrain`

Also should have an xmldoc with the first comment: "Finds the weighted sum of a sequence of strain intervals."